src/main/java/com/suse/saltstack/netapi/parser/JsonParser.java

```diff
@@ -81,4 +89,29 @@ public Date deserialize(JsonElement jsonElement, Type type,
             }
         }
     }
+
+    private class StatsDeserializer implements JsonDeserializer<Stats> {
+
+        @Override
+        public Stats deserialize(JsonElement jsonElement, Type type,
+                JsonDeserializationContext jsonDeserializationContext)
+                throws JsonParseException {
+            try {
+                JsonObject stats = jsonElement.getAsJsonObject();
+                Applications app = gson.fromJson(stats.get("CherryPy Applications"), Applications.class);
+                HttpServer server = null;
+                for(Map.Entry<String, JsonElement> entry : stats.entrySet()){
+                    String key = entry.getKey();
+                    if(key.startsWith("CherryPy HTTPServer ")){
```

Comment:
I would prefer to convert those to constants, even if they are used only once. There is no real advantage of that, it's just a personal preference, I like to keep those strings together in one place. Further in case one of them needs to be adapted slightly, the code can stay untouched.